packages/turf-meta/index.ts

```diff
@@ -50,7 +83,9 @@ function coordEach(geojson, callback, excludeWrapCoord) {
     type = geojson.type,
     isFeatureCollection = type === "FeatureCollection",
     isFeature = type === "Feature",
-    stop = isFeatureCollection ? geojson.features.length : 1;
+    stop = isFeatureCollection
+      ? (geojson as FeatureCollection).features.length
+      : 1;
```

Comment:
This is also an example of another really common change in this PR. TypeScript was not narrowing the types enough and so we require some manual annotation of types to make everything work out. I don't think I'm doing anything in here that isn't being checked at runtime.